app/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/controllers/GlossaryController.scala

```diff
@@ -39,31 +40,31 @@ class GlossaryController @Inject()(mcc: MessagesControllerComponents,
 
   def show: Action[AnyContent] = Action { implicit request =>
     implicit val requestLanguage: Lang = request.lang(messagesApi)
-    val glossaryList: List[(String, List[(String, String)])] = glossaryService.getGlossaryList
+    val glossaryContent: GlossaryContent = glossaryService.getGlossaryContent()
```

Comment:
Introduced a new type,`GlossaryContent`, for simplicity, and renamed parameters to match everywhere.